testdata/project-v4-with-deploy-image/controllers/busybox_controller.go

```diff
@@ -250,6 +256,17 @@ func labelsForBusybox(name string) map[string]string {
 	return map[string]string{"type": "busybox", "busybox_cr": name}
 }
 
+// imageForBusybox returns the image for the resources
+// belonging to the given Busybox CR name.
+func imageForBusybox() (string, error) {
```

Comment:
Wdyt about we explain that it will be gathering the image from the ENV VAR which exists on the manager.go